src/Database/DatabaseManager.php

```diff
@@ -72,6 +72,23 @@ public function createTenantConnection(TenantWithDatabase $tenant)
         $this->app['config']['database.connections.tenant'] = $tenant->database()->connection();
     }
 
+    /**
+     * Create the tenant's host database connection.
+     */
+    public function createHostConnection(TenantWithDatabase $tenant)
+    {
+        $this->originalTemplate = $this->app['config']["database.connections.{$tenant->database()->getTemplateConnectionName()}"];
```

Comment:
```suggestion
        $this->originalTemplate = $this->app['config']['database.connections'][$tenant->database()->getTemplateConnectionName()];
```

Also please add the property to the class